packages/amplify-provider-awscloudformation/lib/upload-appsync-files.js

```diff
@@ -218,11 +220,30 @@ async function uploadAppSyncFiles(context, resourcesToUpdate, allResources, opti
  * should return the same hash.
  */
 async function hashDirectory(directory) {
```

Comment:
This would create a new S3DeploymentRootKey. Wouldn't we also want to update [resource-status](https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-cli/src/extensions/amplify-helpers/resource-status.js) as well to indicate an update is needed?

Reply:
Not at part of this change, we've a separate issue around last push timestamp and hashes.